contracts/mockContracts/MockDGXDex.sol

```diff
@@ -43,91 +33,102 @@ contract MockDGXDEX is Withdrawable {
+        purchasedAmount = amountDgxWei;
+
+        Purchase(success, purchasedAmount, msg.sender, weiPerDgxMg);
         
     }
 
+    event Sell(bool success, uint256 amountWei, address sellerAddress, uint256 weiPerDgxMg);
+
     function sell(uint256 amount,
-                  uint256 block_number,
-                  uint256  nonce,
+                  uint256 blockNumber,
+                  uint256 nonce,
                   uint256 weiPerDgxMg,
                   address signer,
-                  bytes signature) public returns (bool success) {
+                  bytes signature)
+    public
+    returns (bool success) {
 
         uint256 amountWei;
+        uint256 amountDgxWei;
+        uint256 amountMgDgxWei;
+        bool verified;
+        address actualSigner;
 
         if (sigCheck) {
-            bool verified;
-            address actual_signer;
-            
-            (verified, actual_signer) =   
-            verify_signed_price(block_number,
-                                nonce,
-                                weiPerDgxMg,
-                                signer,
-                                signature);
-
+            (verified, actualSigner) = verify_signed_price(blockNumber,
+                                                           nonce,
+                                                           weiPerDgxMg,
+                                                           signer,
+                                                           signature);
             require(verified);
         }
         if (blockCheck) {
-            require((block_number + 5) >= block.number);
+            require((blockNumber + BLOCK_EXPIRATION) >= block.number);
         }
 
-        amountWei = amount * G_TO_MG * weiPerDgxMg / DGX_DECIMALS;
+        amountDgxWei = amount;
+        amountMgDgxWei = amountDgxWei * GRAM_TO_MILLIGRAM;
+        amountWei = amountMgDgxWei * weiPerDgxMg / DGX_DECIMALS;
+
         require(token.transferFrom(msg.sender, this, amount));
         msg.sender.transfer(amountWei);
         
         success = true;
 
-        Sell(success, amountWei);
+        Sell(success, amountWei, msg.sender, weiPerDgxMg);
     }
 
     function verify_signed_price(uint _block_number, uint _nonce, uint _price, address _signer, bytes _signature)
-        public
-        returns (bool _verified, address _actual_signer)
-        {
-            bytes32 _hash;
-            bool _verifies;
-            _hash = hash_price_data(_block_number, _nonce, _price);
-            
-            ECVerifyContract ECVerify = new ECVerifyContract();
-            
-            (_verifies,_actual_signer) = ECVerify.ecrecovery(_hash, _signature);
-            _verified = (_verifies && (_actual_signer == _signer));
-        }
+    public
+    returns (bool _verified, address _actual_signer)
+    {
+        bytes32 _hash;
+        bool _verifies;
+        _hash = hash_price_data(_block_number, _nonce, _price);
```

Comment:
hash_price_data wrong notation.

By the way. indentation and notation are clearly explained in solidity documents.
also u can run a tool - solhint that issues errors regarding indentation...
https://github.com/protofire/solhint

Reply:
ditto. It's digix contract code.